docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,88 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there were three configuration settings for how Mobile UCRs were restored and sent to devices:
```

Comment:
nit: "restored" might be a confusing term for the users. I'd suggest just keeping "were sent to the devices" or "were sent to the commcare mobile app"

Reply:
Addressed in [7a502da](https://github.com/dimagi/commcare-hq/pull/35276/commits/7a502daec0b884a8874e6f6b1b27899eeffb9a0a).